broker/src/main/java/io/zeebe/broker/system/SystemContext.java

```diff
@@ -34,6 +34,9 @@
       "Snapshot period %s needs to be larger then or equals to one minute.";
   private static final String MMAP_REPLICATION_ERROR_MSG =
       "Using memory mapped storage level is currently unsafe with replication enabled; if you wish to use replication, set useMmap flag to false (e.g. ZEEBE_BROKER_DATA_USEMMAP=false)";
+  private static final String MAX_BATCH_SIZE_ERROR_MSG =
+      "Expected to have an append batch size maximum which is non negative and smaller then Integer.MAX_VALUE, but was '%s'.";
```

Comment:
I guess we can write the real value instead of `Integer.MAX_VALUE` :)

Reply:
hm yeah :shrug:

Reply:
Actually I did this because I think it is more readable then 2147483647 and everyone knows what int max is :sweat_smile: But I can change it

Reply:
I would add the value, since I'm not sure everyone knows what Integer.MAX_VALUE is (is it signed? unsigned? is it int32? int64? maybe they're not java devs :man_shrugging:).